pkgs/test/lib/src/runner/browser/static/run_wasm_chrome.js

```diff
@@ -8,14 +8,14 @@
 (async () => {
   // Fetch and compile Wasm binary.
   let data = document.getElementById('WasmBootstrapInfo').dataset;
-  let modulePromise = WebAssembly.compileStreaming(fetch(data.wasmurl));
 
   // Instantiate the Dart module, importing from the global scope.
-  let dart2wasm = await import('./' + data.jsruntimeurl);
-  let dartInstance = await dart2wasm.instantiate(modulePromise, {});
+  let dart2wasmJsRuntime = await import('./' + data.jsruntimeurl);
+  let compiledModule = await dart2wasmJsRuntime.compileStreaming(fetch(data.wasmurl));
```

Comment:
Can we check whether an `instantiate` method exists and choose at runtime which approach to use?

Reply:
That would definitely be ideal, so we can still support 3.5 SDKs.